CONTRIBUTING.md

```diff
@@ -78,6 +78,46 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 
 * Discord - [#jest](https://discordapp.com/channels/102860784329052160/103622435865104384) on [Reactiflux](http://www.reactiflux.com/)
 
+## How to try a development build of Jest in another project
```

Comment:
So maybe lets put this right between CLA and Bugs?
Sure! Fixes are always welcome!